src/VirtoCommerce.Platform.Web/Controllers/Api/DynamicPropertiesController.cs

```diff
@@ -148,7 +149,20 @@ public async Task<ActionResult<DynamicPropertyDictionaryItemSearchResult>> Searc
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
         public async Task<ActionResult> SaveDictionaryItemsAsync([FromBody] DynamicPropertyDictionaryItem[] items)
         {
-            await _dynamicPropertyDictionaryItemsService.SaveDictionaryItemsAsync(items);
+            try
+            {
+                await _dynamicPropertyDictionaryItemsService.SaveDictionaryItemsAsync(items);
+            }
+            catch (ArgumentNullException ex)
+            {
+                return BadRequest(ex.Message);
+            }
+            catch (InvalidCollectionItemException ex)
+            {
+                BadRequest(ex.Message);
+            }
```

Comment:
There is no need to catch different types of exception if we always return BadRequest(ex.Message)